sumologic/provider.go

```diff
@@ -134,22 +138,27 @@ func resolveRedirectURL(accessId string, accessKey string) (string, error) {
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	accessId := d.Get("access_id").(string)
 	accessKey := d.Get("access_key").(string)
+	authJwt := os.Getenv("SUMOLOGIC_AUTHJWT")
 	environment := d.Get("environment").(string)
 	baseUrl := d.Get("base_url").(string)
 	isInAdminMode := d.Get("admin_mode").(bool)
 
 	msg := ""
-	if accessId == "" {
-		msg = "sumologic provider: access_id should be set;"
-	}
-
-	if accessKey == "" {
-		msg = fmt.Sprintf("%s access_key should be set; ", msg)
+	if authJwt == "" {
+		if accessId == "" || accessKey == "" {
+			msg = "sumologic provider: "
```

Comment:
nit: you should remove the trailing space here as otherwise the resulting error message will have double space